### PR TITLE
API: Endpoint for retrieving the list of groups #21

### LINKED ISSUE #21: API: Endpoint for retrieving the list of groups
On the registration screen, the user is presented with a list of active groups e.g. Class 14.

Create an endpoint that will return the list of available groups.

> Note: Since there is no database yet, feel free to hardcode the list of groups just for testing purposes.

image

### PR DESCRIPTION
**Pull request by yukafjaponesa**
# Description

API endpoint of list of groups. No database yet so hardcoded classes 14-16 for now.

Fixes #21 

# How to test?

Use Swagger. localhost:3000/api/documentation 

# Checklist

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ x] I have made corresponding changes to the documentation
- [x] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

File: src/server/api/controllers/groups.controller.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+/* TODO: This is an example controller to illustrate a server side controller.
```
Comment (tpetrina) [inline src/server/api/controllers/groups.controller.js:1]: Remove comment

File: src/server/api/controllers/groups.controller.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+Can be deleted as soon as the first real controller is added. */
+const knex = require('../../config/db');
+const Error = require('../lib/utils/http-error');
+// const moment = require('moment-timezone');
```
Comment (tpetrina) [inline src/server/api/controllers/groups.controller.js:5]: Remove comment

File: src/server/api/routes/groups.router.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
+/* TODO: This is just an example file to illustrate API routing and
```
Comment (tpetrina) [inline src/server/api/routes/groups.router.js:1]: Remove comment

**Review by tpetrina — APPROVED** [review]

**Review by tpetrina — DISMISSED** [review]

**Review by MaherHussain — APPROVED** [review]

**Review by tpetrina — APPROVED** [review]
Great work, ship it 🚢

**Pull request merged**